### PR TITLE
sgml-common 0.6.3 (new formula)

### PR DESCRIPTION
**Pull request by maxim-belkin**

### CODE REVIEW

**Comment by sjackman** [comment]
If `sgml-common` is a dependency of formulae in `core`, it should be added to `core`.

**Comment by maxim-belkin** [comment]
I think the point is to have Linux-only formulae here even if something depends on them from the core, no?

**Comment by maxim-belkin** [comment]
At least, on the main page we have
```
Linux-only formulae that are not dependencies of core formulae
``` 
:)

**Comment by maxim-belkin** [comment]
oh! **not** I missed that

**Comment by sjackman** [comment]
It's so that if the version of a dependency is bumped, its dependents are tested for linkage breakage. Currently that test is done only for dependents in the same repo. Cross-repo deps are not tested for linkage breakage.

File: Formula/sgml-common.rb [hunk 1]
```diff
@@ -5,6 +5,7 @@ class SgmlCommon < Formula
   sha256 "7dc418c1d361123ffc5e45d61f1b97257940a8eb35d0bfbbc493381cc5b1f959"
 
   depends_on "autoconf" => :build
+  depends_on "aclocal" => :build
```
Comment (sjackman) [inline Formula/sgml-common.rb]: `aclocal` is included in `automake`.

**Pull request closed without merging**